app/views/discord_engine/interactions/connect.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.type @resolver.callback.type
+json.data do
+  json.content @resolver.content
+end
```

Comment:
it looks like this structure will be the same for any response that is a simple message, right? I see it duplicated in `app/views/discord_engine/interactions/say_hi.jbuilder` too. Do you think we could DRY it up somehow moving it in the discord engine?

I think we'll want to reuse this structure whenever we want to respond to an interaction with a simple message, right? maybe we can create a jbuilder partial in the engine and call the partial here?